sites/docs/docs/tutorial-gh-pages.mdx

```diff
@@ -234,11 +247,20 @@ jobs:
       - name: Deploy to GitHub Pages
         if: ${{ github.event_name == 'push' && github.ref == 'refs/heads/main' }}
         id: deployment
-        uses: actions/deploy-pages@v1
+        uses: actions/deploy-pages@v4
     
 ```
 
 
+## Config Vite
+Add the following to the `vite.config.js` file to allow Vite to serve correctly from a subpath like `https://yourname.github.io/your-repo-name/`
```

Comment:
```suggestion
Add the following to the `vite.config.js` file to allow Vite to build the website such that it can be served from a subpath like `https://my-username.github.io/vitessce-demo-gh-pages/`
```